wallbox_client.py

```diff
@@ -15,6 +15,10 @@ class WallboxModbusMixin:
     NUM_MODBUS_PORTS = 65536
     last_restart: int = 0
 
+    # This is the what the charger returns for a disconnected state
```

Comment:
Still unclear if 0 means disconnected and 1 means connected, or vice versa.

Reply:
OK, will explain

Reply:
To be honest it is still unclear what a 0 state means and what a 1 state means.

Reply:
I'll try to add a more clear explanation.